<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTGUIVideoTest1Class</class>
 <widget class="QMainWindow" name="QTGUIVideoTest1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>696</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QTGUIVideoTest1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>SelectModel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TrainModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>TrainModel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TargetShow">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TargetShow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Settings">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QPushButton" name="REPLAY">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>REPLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Camera">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已选模型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ModelPath">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>已选视频:</string>
    </property>
   </widget>
   <widget class="QLabel" name="VideoPath">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>610</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>957</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenuy">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuFile">
     <property name="title">
      <string>File</string>
     </property>
     <addaction name="actionOpen"/>
     <addaction name="actionClose"/>
     <addaction name="actionQuit"/>
    </widget>
    <addaction name="menuFile"/>
    <addaction name="actionTool"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuMenuy"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionTool">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTGUIVideoTest1.qrc"/>
 </resources>
 <connections/>
</ui>
